### DISCUSSION TITLE
Datatables on pages other than list

### DISCUSSION THREAD

**Discussion by sammyblackbaron** [Laravel-Backpack/community-forum#619] [Q&A (Help)] [answered]
Hi,

I have a field in my edit crud method that shows a table of user activity. No interactive functionality, just for data:

```
[   // view
    'name' => 'activity-table',
    'type' => 'view',
    'view' => 'partials/activity-table' // basically just an HTML table
],
```

I would really like to make that table use Datatables, styled like the rest of backpack.

Is that possible?

Still using v5 but planning on upgrading to v6 soon!

**Comment by karandatwani92** [comment, marked as answer]
Hey @sammyblackbaron
Yes, you can **style your custom table view** like the rest of the backpack.

1) Check list operation view here `vendor/backpack/crud/src/resources/views/crud/list.blade.php`

2) Take styles sheet cdn links and use the same CSS classes of the `table` tag.

This way you will achieve the same look. More info on styling data tables is [here](https://datatables.net/examples/styling/)

**Reply by sammyblackbaron** [reply]
Brilliant, thanks @karandatwani92
